main.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"context"
+	"log"
+	"net/http"
+	"os"
+	"os/signal"
+	"time"
+
+	"github.com/wizelineacademy/golang-bootcamp-2020/config"
+)
+
+func main() {
+	// Loggers
+	infoLog := log.New(os.Stdout, "INFO\t", log.Ldate|log.Ltime)
+	errorLog := log.New(os.Stderr, "ERROR\t", log.Ldate|log.Ltime|log.Lmicroseconds|log.Lshortfile)
+
+	//Init Config
+	config, err := config.Init(infoLog, errorLog)
+	if err != nil {
+		errorLog.Fatalf("message: unable to initialize config, err: %v", err)
+	}
+	defer config.DB.Close()
+
+	sm := config.InitRoutes()
+	// HTTP Server configuration
+	s := &http.Server{
+		Addr:         *config.Addr,      //set the bind address
+		Handler:      sm,                //set the default handler
+		ReadTimeout:  5 * time.Second,   //max time to read request from the client
+		WriteTimeout: 10 * time.Second,  //max time to write request from the client
+		IdleTimeout:  120 * time.Second, //max time for connections using TCP Keep-Alive
```

Comment:
5, 10, and 120 are magic numbers.

Define them as part of the configuration (or as constants).